specification/ml/_types/Datafeed.ts

```diff
@@ -17,35 +17,17 @@
  * under the License.
  */
 
+import { OverloadOf } from '@spec_utils/behaviors'
 import { Dictionary } from '@spec_utils/Dictionary'
 import { AggregationContainer } from '@_types/aggregations/AggregationContainer'
-import { ExpandWildcards, Id, Indices, IndicesOptions } from '@_types/common'
+import { Id, IndexName, Indices, IndicesOptions } from '@_types/common'
 import { RuntimeFields } from '@_types/mapping/RuntimeFields'
 import { double, integer, long } from '@_types/Numeric'
 import { QueryContainer } from '@_types/query_dsl/abstractions'
 import { ScriptField } from '@_types/Scripting'
 import { Time, Timestamp } from '@_types/Time'
 import { DiscoveryNode } from './DiscoveryNode'
 
-export class Datafeed {
```

Comment:
Should we use `Datafeed` or `DatafeedConfig` for this type? Everywhere where users define a datafeed it's using `datafeed_config` but when we interact with the datafeed itself it's not called "config", just "datafeed". Maybe should change this to `Datafeed`?